modules/standard/Regexp.chpl

```diff
@@ -451,6 +451,11 @@ class BadRegexpError : Error {
                    ``(?U)``.
 
  */
+
+/*
+  throws: SystemError: If CHPL_REGEXP is set to none.
+*/
```

Comment:
You'll want to include this as part of the previous comment - putting it here will prevent the other documentation from being include (and it's useful :) )